### PR TITLE
Add SsisProperties if NodeCount not null for managed ir

### PR DESCRIPTION
**Pull request by HaoQian-MS**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by maddieclayton** [comment]
@azuresdkci Add to whitelist

**Comment by maddieclayton** [comment]
@HaoQian-MS Please add a changelog entry for this change, then it should be good to go.

File: ChangeLog.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 #### Az.DataFactory
 * Updated ADF .Net SDK version to 3.0.2
 * Updated Set-AzDataFactoryV2 cmdlet with extra parameters for RepoConfiguration related settings.
+* Add SsisProperties if NodeCount not null for managed integration runtime.
```
Comment (maddieclayton) [inline ChangeLog.md]: Please remove this, this will be updated next time we do a stable Az release.

**Pull request merged**